### ISSUE TITLE
getSourceEditorContext not found in RStudio

### ISSUE THREAD

**Issue by crsh** [rstudio/rstudioapi#24]
I'm trying to use `getSourceEditorContext()` and `getConsoleEditorContext()` but both functions throw errors like this.

~~~{r}
> rstudioapi::getSourceEditorContext()
Error: Function getSourceEditorContext not found in RStudio
~~~

`getActiveDocumentContext()` works as expected. Am I missing something?

~~~{r}
> devtools::session_info()
Session info ---------------------------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.3.1 (2016-06-21)
 system   x86_64, linux-gnu           
 ui       RStudio (0.99.903)          
 language de_DE                       
 collate  de_DE.UTF-8                 
 tz       <NA>                        
 date     2016-09-07                  

Packages -------------------------------------------------------------------------------------------------------------------------
 package    * version date       source                             
 devtools     1.11.1  2016-04-21 CRAN (R 3.3.0)                     
 digest       0.6.10  2016-08-02 CRAN (R 3.3.1)                     
 memoise      1.0.0   2016-01-29 CRAN (R 3.3.0)                     
 rsconnect    0.4.3   2016-05-02 CRAN (R 3.3.0)                     
 rstudioapi   0.6     2016-09-07 Github (rstudio/rstudioapi@55d3200)
 withr        1.0.1   2016-02-04 CRAN (R 3.3.0)     
~~~

**Comment by kevinushey** [comment]
You likely need to update to the preview release of RStudio: https://www.rstudio.com/products/rstudio/download/preview/

**Comment by crsh** [comment]
According to the documentation both functions are supposed to be available as of version 0.99.1111.

**Comment by crsh** [comment]
Ah sorry, I just now realized that 1111 > 903. My bad. :bow:
